.github/workflows/ci.yml

```diff
@@ -32,6 +32,12 @@ on:
         description: |
           Fail pipeline if CVEs are identified.
           Default is '0', possible values '0' or '1'
+      cve-issue-repo:
+        type: string
+        required: false
+        default: ${{ github.repository }}
```

Comment:
Let's remove this config - repo for issue is already configured by `${{ vars.GH_ORGANIZATION }}` and `${{ vars.GH_ISSUES_REPO }}`.

See `notify_failure:` and `close_issue:` for more details.

Reply:
Where are `GH_ISSUES_REPO` and `GH_ORGANIZATION` coming from?

Reply:
Org-level variables.